fluss-filesystems/fluss-fs-hadoop/pom.xml

```diff
@@ -46,8 +46,9 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-hdfs</artifactId>
-            <!-- to keep same to the haddop version of fluss-fs-hadoop-shaded -->
+            <!-- to keep same to the hadoop version of fluss-fs-hadoop-shaded -->
             <version>${fs.hadoopshaded.version}</version>
+            <scope>provided</scope>
```

Comment:
The `hadoop-hdfs` will still be needed for access hadoop filsystem via our `fluss-fs-hadoop` lib. I rememer at first, the  dependency is with `<optional>true</optional>`, but then we want to access hadoop filsystem via our `fluss-fs-hadoop` lib directly, so I remove the `<optional>true</optional>` to make it include in `hadoop-hdfs` lib.

Maybe we can left a todo to include it again in #837

Reply:
OK. Let's add it back into `fluss-fs-hdfs` in #837.